### PR TITLE
v2021-02-15 Post-release 

### PR DESCRIPTION
**Pull request by rvl**
# Issue Number

ADP-712

# Overview

Update version numbers, release notes, etc, after tagging of v2021-02-15.

### CODE REVIEW

File: .github/RELEASE_TEMPLATE.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ Compatible with [`cardano-node@{{CARDANO_NODE_TAG}}`](https://github.com/input-o
 
 <!-- A snapshot of the documentation at the time of releasing. -->
 
-[API Documentation](https://input-output-hk.github.io/cardano-wallet/api/{{GIT_TAG}})
-[CLI Manual](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-command-line-interface/{{WIKI_COMMIT}})
-[Docker Manual](https://github.com/input-output-hk/cardano-wallet/wiki/Docker/{{WIKI_COMMIT}})
+| :closed_book: | :computer: | :whale: |
+| -- | -- | -- |
+| [API Documentation](https://input-output-hk.github.io/cardano-wallet/api/{{GIT_TAG}}) | [CLI Manual](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-command-line-interface/{{WIKI_COMMIT}}) | [Docker Manual](https://github.com/input-output-hk/cardano-wallet/wiki/Docker/{{WIKI_COMMIT}}) |
```
Comment (KtorZ) [inline .github/RELEASE_TEMPLATE.md:33]: :tada:

**Review by KtorZ — APPROVED** [review]

**Comment by KtorZ** [comment]
bors retry

**Review by piotr-iohk — APPROVED** [review]

**Pull request merged**